_proposals/pup-0001.rst

```diff
@@ -0,0 +1,317 @@
+---
+title: PUP Purpose and Guidelines
+author: Brian Bouterse
+---
+
+Summary
+=======
+
+This proposal provides a structured process to propose, track, discuss, and decide on process
+changes for the Pulp project and community.
+
+Motivation
+==========
+
+As initially [raised on the pulp-dev list](https://www.redhat.com/archives/pulp-dev/2017-February/msg00014.html),
+Pulp lacks a process to propose, track, discuss, and decide on changes that affect the Pulp
+community.
+
+This is a problem that has been addressed by many communities. The process outlined in this document
+is inspired by processes in the Python, Django, and Foreman communities.
+
+
+What are these called?
+======================
+
+We need an acronym or initialization for these. Here are some quick ideas:
+
+* "Pulp Update Proposal" (PUP)
+* "Pulp Change Proposal" (PCP)
+* <Your idea here>
+
+
+Tracking Proposals
+==================
+
+Proposals would be in restructured text format and sent as a pull request against the
+pulpproject.org repo or a dedicated repo. We could easily create a tab at the top of pulpproject.org
+to show off merged proposals. This could also be in a separate, dedicated repo like Python does, but
```

Comment:
I suspect the simplest contributor experience would happen if there is a dedicated repo, a README at the root that summarizes how things work, and users can make PRs against master. That makes the "source" very easy to find and understand. I'm ok with any arrangement though.